pkg/config/config.go

```diff
@@ -548,6 +548,8 @@ func initConfig(config Config) {
 	config.SetKnown("apm_config.receiver_timeout")
 	config.SetKnown("apm_config.watchdog_check_delay")
 
+	config.BindEnvAndSetDefault("inventories_enabled", false)
```

Comment:
Should we not enable it by default ?

Reply:
It's a beta thing, so I assumed not.

Reply:
@ItsYamnitsky What should be the default ?